ravenframework/SupervisedLearning/SupervisedLearning.py

```diff
@@ -335,9 +335,9 @@ def evaluate(self,edict):
     names, values  = list(edict.keys()), list(edict.values())
     stepInFeatures = 0
     for index in range(len(values)):
-      #If value is a float, convert to numpy array for evaluation
-      if type(values[index]).__name__ == 'float':
-        values[index] = np.array(values[index])
+      # If value is a float or int, convert to numpy array for evaluation
+      if isinstance(values[index], (float, int, np.number)):
```

Comment:
```suggestion
      if mathUtils.isAFloatOrInt(values[index]):
```

Added this suggesting since that check you are doing here is == of what ``mathUtils.isAFloatOrInt`` does